compose-builder/Makefile

```diff
@@ -130,14 +156,29 @@ ifeq (ds-random, $(filter ds-random,$(ARGS)))
 		else
 			TOKEN_LIST:=$(TOKEN_LIST),device-random
 		endif
+		ifeq ($(KNOWN_SECRETS_LIST),"")
+			KNOWN_SECRETS_LIST:=redisdb[device-random]
+		else
+			KNOWN_SECRETS_LIST:=$(KNOWN_SECRETS_LIST),redisdb[device-random]
+		endif
 		extension_file:= $(shell GEN_EXT_DIR="$(GEN_EXT_DIR)" ./gen_secure_compose_ext.sh device-random)
 		COMPOSE_FILES:=$(COMPOSE_FILES) -f $(extension_file)
 	endif
 endif
 ifeq (ds-rest, $(filter ds-rest,$(ARGS)))
 	COMPOSE_FILES:=$(COMPOSE_FILES) -f add-device-rest.yml
 	ifneq (no-secty, $(filter no-secty,$(ARGS)))
-		extension_file:= $(shell GEN_EXT_DIR="$(GEN_EXT_DIR)" ./gen_secure_compose_ext.sh device-rest device-rest device-rest-go)
+		ifeq ($(TOKEN_LIST),"")
+			TOKEN_LIST:=edgex-device-rest
+		else
+			TOKEN_LIST:=$(TOKEN_LIST),edgex-device-rest
+		endif
```

Comment:
prefer to fix the default list in edgex-go

Reply:
We have service key naming inconsistency across device service that I want to address and hope to fix the Device Rest key to match want is in edgex-go. Until then when need to address it here in the compose files.
I will add TODO comment to address it.

Reply:
sounds good; for the time being is ok and eventual consistency is the goal so that it won't cause any troubles or misleading latter on.